docs/releases/v1.33.0.md

```diff
@@ -0,0 +1,267 @@
+    ```
+  
+    ```yaml
+    # no taints
+    spec:
+      kubernetes:
+        masters:
+          taints: []
+    ```
+
+- [[#435](https://github.com/sighupio/distribution/pull/435)] Repository management lifecycle configuration for OnPremises provider:
+  - Added new boolean configuration fields for environments where package repositories are configured outside of furyctl.
+    - `spec.kubernetes.loadBalancers.manageRepositories`: Controls HAProxy repository setup
+    - `spec.kubernetes.advanced.containerd.manageRepositories`: Controls NVIDIA container toolkit's repository setup
+    - `spec.kubernetes.advanced.manageRepositories`: Controls Kubernetes package repository setup
+  - All fields are optional. If omitted, the system defaults to automatic repository management.
+
+    ```yaml
+    spec:
+      kubernetes:
+        loadBalancers:
+          enabled: true
+          manageRepositories: false
+        advanced:
+          manageRepositories: false
+          containerd:
+            manageRepositories: false
+    ```
+
+- [[#353](https://github.com/sighupio/fury-distribution/pull/353)] **Add EKS self-managed node pool default override options for IDMS**: add a variable to override the default properies for EKS self-managed node pools. Currently support only the IDMS ones.
+
+## Fixes 🐞
+
+- [installer-eks/issues#88](https://github.com/sighupio/installer-eks/issues/88) This PR fixes an issue when using `selfmanaged` nodes with `alinux2023`. The way we used to provision images relied on Amazon's `bootstrap.sh` which has been deprecated in favor of `nodeadm`.
+
+- Plugins names are now pattern-validated in the schema to avoid potential errors at runtime when setting invalid names.
+
+### Security fixes
+
+## Upgrade procedure
+Check the [upgrade docs](https://docs.sighup.io/docs/installation/upgrades/) for the steps to upgrade the SIGHUP Distribution from one versions to the next using furyctl.
+
+In particular, if using an EKS provider, you first need to migrate your nodePools from `alinux2` to `alinux2023` before upgrading to v1.33. :warning: There might be downtime as new pools get replaced.
+1. Update furyctl to version v0.33.0 (if you fail to update, aws-load-balancer-controller will fail in a CrashLoopBackOff state when upgrading to alinux2023 in eks-managed nodePools in v1.32.0)
+2. :warning: Do not upgrade yet! While still on v1.32, migrate the node pools:
+    - for `eks-managed` nodePools, replace your current `.spec.kubernetes.nodePools[*].ami.type: alinux2` nodePool with a new `.spec.kubernetes.nodePools[*].ami.type: alinux2023` nodePool
+    - for `self-managed` nodePools, replace the AMI ID or `.spec.kubernetes.nodePoolGlobalAmiType` to `alinux2023`.
+3. Upgrade to v1.33
+
+:warning: If you don't follow the above guide and upgrade AND migrate nodePools at the same time, you will face Terraform errors hard to fix and a broken cluster. Please first migrate your nodePools and then upgrade to v1.33.
```

Comment:
let's use an [admonition](https://github.com/orgs/community/discussions/16925) that is more visibly highlighted:

```suggestion
> [!WARNING]
> If you don't follow the above guide and upgrade AND migrate nodePools at the same time, you will face Terraform errors hard to fix and a broken cluster. Please first migrate your nodePools and then upgrade to v1.33.
```